### DISCUSSION TITLE
DevTools 2.0 ideas

### DISCUSSION THREAD

**Discussion by RachelChanX** [DevToys-app/DevToys#911] [Ideas]
I tried to send it in the DevTool 2.0 Form but it reached maximum answers so I will just post it here

I wish it would be my only place for tools because right now many features and tools not included 

For example:
1. I want to be able to convert Java class fields to JSON

2. Generate Random JSON by fields
something like this
https://www.mockaroo.com/

3. Eyedrop with that generate lighter and darker shades base on the color picked

4. In all tool page you should add recent used and also add favorites tab

5. Maybe you can consider "About" tab with link to latest version feature list and the current app version and latest time updated etc.

I hope you will read it and take some of it in to consideration

**Comment by maroonedsia** [comment]
Here is another idea: please expose the tools to a CLI, if applicable. The GUI could use the same APIs internally. 
Having these tools over a CLI could hugely benefit many automation and CI/CD scenarios.

**Reply by veler** [reply]
This will be supported in DevToys 2.0. It's in development.

**Comment by maroonedsia** [comment]
And another one: Allow plugins! I have created several small dev tools for myself. If you let developers add community plugins to this tool, it'll take the DevToys to the next level.

**Reply by veler** [reply]
This will be supported too in DevToys 2.0.
